server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksService.scala

```diff
@@ -64,23 +77,32 @@ class StorageLinksService(storageLinks: StorageLinksCache, googleStorageAlg: Goo
     } yield link
   }
 
+  private def persistWorkspaceBucket(baseDirectory: LocalDirectory, cloudStorageDirectory: CloudStorageDirectory): IO[Unit] = {
+    val fileBody =
+      RuntimeVariables(s"gs://${cloudStorageDirectory.bucketName.value}/notebooks") //appending notebooks to mimick old jupyter image until we start using new images.
```

Comment:
This is to make it work with the current logic in `kernel-bootstrap.sh`?

Reply:
yes

Reply:
Also I thought the old format of `.delocalize.json` was something like:
```
{
  "destination": "gs://foo/notebooks"
}
```
Does this output the same format?

Reply:
Ah nvm I see `runtimeVariablesEncoder` -- looks good.